src/main.js

```diff
@@ -308,6 +308,9 @@ class MiniClimate extends LitElement {
         item.functions.icon.style = compileTemplate(item.icon.style, context);
     }
 
+    if (item.disabled)
```

Comment:
lets add support for boolean as well, and extend this ability to other hide functionality in other componenets

Reply:
How do you handle those ?

Reply:
You can check the type of the value given, if it is boolean, just return a function that returns that value